rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/ModerationRequestController.java

```diff
@@ -116,26 +124,36 @@ public ResponseEntity<CollectionModel<ModerationRequest>> getModerationRequests(
         return getModerationResponseEntity(pageable, request, allDetails, modRequestsWithPageData);
     }
 
-    @Operation(
-            summary = "Get a single moderation request.",
-            description = "Get a single moderation request by id.",
-            tags = {"Moderation Requests"}
-    )
+    @Operation(summary = "Get a single moderation request.", description = "Get a single moderation request by id.", tags = {
+            "Moderation Requests" })
     @RequestMapping(value = MODERATION_REQUEST_URL + "/{id}", method = RequestMethod.GET)
-    public ResponseEntity<HalResource<ModerationRequest>> getModerationRequestById(
-            @Parameter(description = "The id of the moderation request to be retrieved.")
-            @PathVariable String id
-    ) throws TException {
+    public ResponseEntity getModerationRequestById(
```

Comment:
Any reason to remove the object type from `ResponseEntity`? It causes wrong Spring documents to be generated.

Reply:
Added proper Object type.